### PR TITLE
[FSSDK-8963] doc: Full Stack to Feature Experimentation Rename

### PR DESCRIPTION
**Pull request by mikechu-optimizely**
## Summary
- Update instances of Full Stack to Feature Experimentation
- Reformat README
- Add packagist badge

## Test plan

- Existing unit tests & FSC e2e tests should pass

## Issues
-FSSDK-8963

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,38 +1,43 @@
 # Optimizely PHP SDK
+[![Packagist](https://badgen.net/packagist/v/optimizely/optimizely-sdk)](https://packagist.org/packages/optimizely/optimizely-sdk)
 [![Build Status](https://travis-ci.org/optimizely/php-sdk.svg?branch=master)](https://travis-ci.org/optimizely/php-sdk)
```
Comment (russell-loube-optimizely) [inline README.md:3]: Nit: take same approach as other SDK (Flutter?) and remove build badge until it's not failing?

File: composer.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "optimizely/optimizely-sdk",
-  "description": "Optimizely SDK for Full Stack PHP projects.",
+  "description": "Optimizely PHP SDK for Optimizely Feature Experimentation, Optimizely Full Stack (legacy), and Optimizely Rollouts.",
```
Comment (russell-loube-optimizely) [inline composer.json:3]: Nit:

```suggestion
  "description": "Optimizely PHP SDK for Optimizely Feature Experimentation, Optimizely Full Stack (legacy), and Optimizely Rollouts",
```

**Review by russell-loube-optimizely — COMMENTED** [review]
LGTM nice job dogg.

**Review by jaeopt — APPROVED** [review]
LGTM

**Pull request merged**